apps/cli-daemon/src/app/session/session.service.ts

```diff
@@ -0,0 +1,14 @@
+import { Injectable } from '@nestjs/common';
+
+@Injectable()
+export class SessionService {
+  private _cwd = __dirname;
```

Comment:
you sure its not supposed to be `process.cwd()`? https://stackoverflow.com/a/45145514/6101529

Reply:
it doesn't really matter in our case, but I don't mind changing it.